microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -69,25 +69,25 @@ Mobile devices have much less CPU power than desktops and laptops.  Whenever you
 
    ![CPU throttle.](../media/evaluate-performance-performance-capture-settings.msft.png)
 
```

Comment:
Change:
Click **Capture Settings** 
to:
Click **Capture settings**

tooltip is lowercase

Reply:
Fixed in latest commit; thanks!